pytket/extensions/stim/backends/stim_backend.py

```diff
@@ -63,6 +65,14 @@
     OpType.Measure: "M",
     OpType.Reset: "R",
 }
+_backend_info = BackendInfo(
+    "StimBackend",
+    None,
+    __extension_version__,
+    None,
+    None,
```

Comment:
Is there one too many `None` here? (Looks like the order is: version, architecture, gate set.)